src/App.js

```diff
@@ -1,18 +1,39 @@
 import React from 'react';
+const programTitle = "Todo List";
+const todoList = [
```

Comment:
Small nitpick for these const data type variables (**doesn't require any changes**):

When there are `const` data type variables -- boolean, string, array and number -- that live outside of a React function, you'll often find them as UPPERCASE to signify they'll never change. 

Therefore, `projectList` would become `PROJECT_LIST` to signify it's a static constant that should never be changed.

Once again, this **doesn't require any changes**, so you're welcomed to keep your code as is.